### ISSUE TITLE
chessrunbot: White king not found (gameid 1566 (previously 1448))

### ISSUE THREAD

**Issue by ArchieT** [ArchieT/3manchess#30]
```
 platinum  archiet  ⋯  3manchess  cmd  chessrunbot   master  chessrunbot -login remotetest -passwd remotetest -botid 1 -color 1 -gameid 1448
2016/04/08 14:43:20 1
2016/04/08 14:43:20 White
2016/04/08 14:43:20 1 [56 48 98 49 56 101 53 99 52 52 99 54 49 56 56 102 53 99 53 49 50 55 57 100 98 98 99 52 55 54 51 99]
2016/04/08 14:43:20 botid 1
2016/04/08 14:43:20 &{[99 111 110 115 116 115 105 116 118 97 108 45 100 101 109 111 116 101 115 116] 1 bot0 2 [123 34 68 101 112 116 104 34 58 48 44 34 79 119 110 101 100 84 111 84 
104 114 101 97 116 101 110 101 100 34 58 52 125]}
2016/04/08 14:43:20 White
2016/04/08 14:43:20 &{0xc208067040 1448 0xc20809f400 0xc208064d80 1 {2 [99 97 102 102 54 56 56 97 98 56 97 101 57 50 49 97 55 54 49 102 54 101 100 56 52 50 101 98 56 57 49 100]} 
0xc208013020 0x40b5b8}
panic: King not found!!!: White

goroutine 33 [running]:
github_com_ArchieT_3manchess_game.CheckChecking.pN39_github_com_ArchieT_3manchess_game.Board
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/move.go:174
github_com_ArchieT_3manchess_game.AmIInCheck.pN39_github_com_ArchieT_3manchess_game.State
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/endgame.go:35
github_com_ArchieT_3manchess_game.After.pN38_github_com_ArchieT_3manchess_game.Move
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/move.go:429
github_com_ArchieT_3manchess_game.CanIMoveWOCheck.pN39_github_com_ArchieT_3manchess_game.State
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/endgame.go:13
github_com_ArchieT_3manchess_game.EvalDeath.pN39_github_com_ArchieT_3manchess_game.State
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/state.go:224
github_com_ArchieT_3manchess_game.EvalAfter.pN38_github_com_ArchieT_3manchess_game.Move
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/move.go:440
game.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/asaomgen.go:20
created by github_com_ArchieT_3manchess_game.ASAOMGen
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/asaomgen.go:8

goroutine 16 [chan receive]:
main.main
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/cmd/chessrunbot/main.go:136

goroutine 18 [finalizer wait]:

goroutine 25 [chan receive]:
main.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/cmd/chessrunbot/main.go:88
created by main.main
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/cmd/chessrunbot/main.go:87

goroutine 23 [IO wait]:

goroutine 24 [select]:

goroutine 26 [chan receive]:
constsitval.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/ai/constsitval/ai.go:72
created by github_com_ArchieT_3manchess_ai_constsitval.Initialize.pN52_github_com_ArchieT_3manchess_ai_constsitval.AIPlayer
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/ai/constsitval/ai.go:71

goroutine 27 [chan receive]:
remote.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:57
created by github_com_ArchieT_3manchess_client_remote.New
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:56

goroutine 28 [chan receive]:
github_com_ArchieT_3manchess_ai_constsitval.Worker.pN52_github_com_ArchieT_3manchess_ai_constsitval.AIPlayer
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/ai/constsitval/ai.go:92
github_com_ArchieT_3manchess_ai_constsitval.Think.pN52_github_com_ArchieT_3manchess_ai_constsitval.AIPlayer
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/ai/constsitval/ai.go:144
github_com_ArchieT_3manchess_ai_constsitval.HeyItsYourMove.pN52_github_com_ArchieT_3manchess_ai_constsitval.AIPlayer
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/ai/constsitval/ai.go:154
github_com_ArchieT_3manchess_client_remote.askplayer.pN44_github_com_ArchieT_3manchess_client_remote.G
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:68
github_com_ArchieT_3manchess_client_remote.Turn.pN44_github_com_ArchieT_3manchess_client_remote.G
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:150
github_com_ArchieT_3manchess_client_remote.Procedure.pN44_github_com_ArchieT_3manchess_client_remote.G
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:161
created by github_com_ArchieT_3manchess_client_remote.New
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/client/remote/remote.go:61

goroutine 29 [chan receive]:
simple.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:11
created by github_com_ArchieT_3manchess_simple.MergeBool
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:18

goroutine 30 [chan receive]:
simple.$nested0
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:11
created by github_com_ArchieT_3manchess_simple.MergeBool
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:18

goroutine 31 [semacquire]:
simple.$nested1
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:20
created by github_com_ArchieT_3manchess_simple.MergeBool
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/simple/merge.go:20

goroutine 32 [chan send]:
game.$nested1
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:21
created by github_com_ArchieT_3manchess_game.VFTPGen
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:8

goroutine 34 [chan send]:
game.$nested1
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:21
created by github_com_ArchieT_3manchess_game.VFTPGen
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:8

goroutine 46 [runnable]:
game.$nested1
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:21
created by github_com_ArchieT_3manchess_game.VFTPGen
        /home/archiet/gopath/src/github.com/ArchieT/3manchess/game/vftpgen.go:8
 platinum  archiet  ⋯  3manchess  cmd  chessrunbot   master  2 
```

**Comment by ArchieT** [comment]
```
 platinum  archiet  ⋯  3manchess  cmd  chessbefores   master  2  chessbefores -gameid=1448
1448
&{1448 0xc2080013a8 0xc2080013b8 0xc2080013c8 2016-04-08 07:10:13 +0000 UTC}

[__ .N .B .Q .K .B .N .R .♖ __ .♗ .♕ .♔ .♗ .♘ .♖ __ .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[.R __ __ __ __ __ __ __ __ __ .♘ __ __ __ __ __ .r __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] White [[true false] [true true] [true false]] [[127,127] [127,127]] 3 13 [true true true]
 <nil>
[{1409 {[0 16 2 16] 1447 1448 0 2}}]
1447
&{1447 0xc2080987f8 0xc208098808 0xc208098818 2016-04-08 07:09:01 +0000 UTC}

[__ .N .B .Q .K .B .N .R .♖ __ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[.R __ __ __ __ __ __ __ __ __ .♘ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Black [[true false] [true true] [true true]] [[127,127] [127,127]] 2 12 [true true true]
 <nil>
[{1408 {[0 9 2 10] 1446 1447 0 2}}]
1446
&{1446 0xc2080018b8 0xc2080018d8 0xc208001908 2016-04-08 07:08:04 +0000 UTC}

[__ .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[.R __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Gray [[true false] [true true] [true true]] [[127,127] [127,127]] 1 11 [true true true]
 <nil>
[{1407 {[0 0 2 0] 1445 1446 0 2}}]
1445
&{1445 0xc208099108 0xc208099118 0xc208099128 2016-04-08 07:07:23 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] White [[true true] [true true] [true true]] [[127,127] [127,127]] 0 10 [true true true]
 <nil>
[{1406 {[3 16 4 16] 1444 1445 0 2}}]
1444
&{1444 0xc208086058 0xc208086068 0xc208086078 2016-04-08 07:06:37 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Black [[true true] [true true] [true true]] [[127,127] [127,127]] 0 9 [true true true]
 <nil>
[{1405 {[3 8 4 7] 1443 1444 0 2}}]
1443
&{1443 0xc208087fc8 0xc208087fd8 0xc208087fe8 2016-04-08 07:05:59 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Gray [[true true] [true true] [true true]] [[127,127] [127,127]] 0 8 [true true true]
 <nil>
[{1404 {[3 0 4 0] 1442 1443 0 2}}]
1442
&{27 0xc208086448 0xc208086458 0xc208086468 2016-04-08 07:05:21 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] White [[true true] [true true] [true true]] [[127,127] [127,127]] 0 7 [true true true]
 <nil>
[{1403 {[2 16 3 16] 1441 1442 0 2}}]
1441
&{26 0xc2080982d8 0xc2080982e8 0xc208098308 2016-04-08 07:04:44 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Black [[true true] [true true] [true true]] [[127,127] [127,127]] 0 6 [true true true]
 <nil>
[{1402 {[2 8 3 8] 1440 1441 0 2}}]
1440
&{25 0xc208086628 0xc208086638 0xc208086648 2016-04-08 07:04:13 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[.P __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Gray [[true true] [true true] [true true]] [[127,127] [127,127]] 0 5 [true true true]
 <nil>
[{1401 {[2 0 3 0] 1439 1440 0 2}}]
1439
&{24 0xc208098488 0xc208098498 0xc2080984a8 2016-04-08 07:03:44 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ __ .p .p .p .p .p .p .p]
[.P __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ .p __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] White [[true true] [true true] [true true]] [[127,127] [127,127]] 0 4 [true true true]
 <nil>
[{1400 {[1 16 2 16] 1438 1439 0 2}}]
1438
&{23 0xc208086538 0xc208086548 <nil> 2016-04-08 07:03:04 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P __ .♙ .♙ .♙ .♙ .♙ .♙ .♙ .p .p .p .p .p .p .p .p]
[.P __ __ __ __ __ __ __ .♙ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Black [[true true] [true true] [true true]] [[127,127] [127,127]] 0 3 [true true true]
 <nil>
[{1399 {[1 8 2 8] 1437 1438 0 2}}]
1437
&{14 0xc2080d3848 <nil> <nil> 2016-04-08 07:02:27 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[__ .P .P .P .P .P .P .P .♙ .♙ .♙ .♙ .♙ .♙ .♙ .♙ .p .p .p .p .p .p .p .p]
[.P __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] Gray [[true true] [true true] [true true]] [[127,127] [127,127]] 0 2 [true true true]
 <nil>
[{1398 {[1 0 2 0] 1436 1437 0 2}}]
1436
&{10 <nil> <nil> <nil> 2016-04-08 07:01:51 +0000 UTC}

[.R .N .B .Q .K .B .N .R .♖ .♘ .♗ .♕ .♔ .♗ .♘ .♖ .r .n .b .q .k .b .n .r]
[.P .P .P .P .P .P .P .P .♙ .♙ .♙ .♙ .♙ .♙ .♙ .♙ .p .p .p .p .p .p .p .p]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
[__ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __ __]
 [false false false] White [[true true] [true true] [true true]] [[127,127] [127,127]] 0 1 [true true true]
 <nil>
[]
 platinum  archiet  ⋯  3manchess  cmd  chessbefores   master 

```

**Comment by Subc2** [comment]
Solved in 1389e14aa170dc7e49a0ef9ae7282deb807ff3f8, tested in 3e364e749b8e7ae52044f2c83415a5860fd264d7.

**Comment by ArchieT** [comment]
```
 mf  ⋯  3manchess  client  remote   master  130  chessrunbot -botid 2 -login remotetest -passwd remotetest -color 1 -gameid 1566
2016/04/10 17:38:29 1
2016/04/10 17:38:29 White
2016/04/10 17:38:30 1 [56 48 98 49 56 101 53 99 52 52 99 54 49 56 56 102 53 99 53 49 50 55 57 100 98 98 99 52 55 54 51 99]
2016/04/10 17:38:30 botid 2
2016/04/10 17:38:30 &{[99 111 110 115 116 115 105 116 118 97 108 45 100 101 109 111 116 101 115 116] 1 bot1 3 [123 34 68 101 112 116 104 34 58 48 44 34 79 119 110 101 100 84 111 84 104 114 101 97 116 101 110 101 100 34 58 53 125]}
2016/04/10 17:38:30 White
2016/04/10 17:38:30 &{0xc8200c6a60 1566 0xc8200fa4e0 0xc820110340 1 {3 [98 55 97 50 100 50 52 55 56 101 51 55 99 50 97 102 51 102 51 50 48 97 97 54 54 98 48 101 98 57 100 48]} 0xc8201285a0 0x4033f0}
panic: King not found!!!: White

goroutine 88 [running]:
panic(0x6e7c00, 0xc8201d9cb0)
	/usr/lib/go/src/runtime/panic.go:464 +0x3e6
github.com/ArchieT/3manchess/game.(*Board).CheckChecking(0xc82024a6c0, 0xc801010101, 0xc801000000)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/move.go:174 +0x1c9
github.com/ArchieT/3manchess/game.(*State).AmIInCheck(0xc82010c880, 0xc820000101, 0xc820000000)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/endgame.go:35 +0x4b
github.com/ArchieT/3manchess/game.(*Move).After(0xc820161da8, 0x0, 0x0, 0x0)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/move.go:429 +0x4be
github.com/ArchieT/3manchess/game.(*State).CanIMoveWOCheck(0xc82010c860, 0x452001, 0xc8204561e0)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/endgame.go:13 +0xfc
github.com/ArchieT/3manchess/game.(*State).EvalDeath(0xc82010c860)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/state.go:224 +0x2e
github.com/ArchieT/3manchess/game.(*Move).EvalAfter(0xc820161f30, 0xc82010c860, 0x0, 0x0)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/move.go:440 +0x6a
github.com/ArchieT/3manchess/game.ASAOMGen.func1(0xc82010db20, 0xc82004b901, 0xc8204561e0)
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/asaomgen.go:20 +0x518
created by github.com/ArchieT/3manchess/game.ASAOMGen
	/home/mf/gopath/src/github.com/ArchieT/3manchess/game/asaomgen.go:27 +0x66
 mf  ⋯  3manchess  client  remote   master  2  
```
